doc/develop/test/bsim.rst

```diff
@@ -150,23 +158,33 @@ required, and some Zephyr users do not use or have west, but still use the build
 Test scripts
 ------------
 
-- Each test is defined by a shell script with the extension ``.sh``.
-- Scripts starting with an underscore (``_``) are ignored.
-- Test scripts expect that the binaries they require are already built, and will spawn the processes
-  for the simulated devices and physical layer simulation with the necessary command line options.
-- Tests must return 0 to the invoking shell if the test passes, and not 0 if the test fails.
-- It is recommended to have a single test for each test script.
+Please follow the conventions and not design one-off bespoke runners (e.g. a python script, or
+another shell abstraction).
+
+The rationale is that it is easier and faster for the maintainers to perform tree-wide updates for
+build system or compatibility changes if the tests are run in the same manner, with the same
+variables, etc..
+
+If you have a good idea for improving your test script, please make a PR changing _all_ the test
```

Comment:
> please make a PR changing 

maybe better:
"please bring it up in the babblesim discord channel, or create an RFC issue or PR"
(no need to demand somebody starts by doing all the work, they can start asking or just proposing the change)

Reply:
yeah ofc

Reply:
but also everybody has grand ideas and I'd like to avoid endless design-by-committee if possible.